{"name": "test_simple_assume7", "status": "failed", "statusDetails": {"message": "pytest_assume.plugin.FailedAssumption: \n2 Failed Assumptions:\n\ntest_xdist.py:36: AssumptionFailure\n>>\tpytest.assume(1==0)\nAssertionError: assert False\n\ntest_xdist.py:38: AssumptionFailure\n>>\tpytest.assume(False)\nAssertionError: assert False", "trace": "tp = <class 'pytest_assume.plugin.FailedAssumption'>, value = None, tb = None\n\n    def reraise(tp, value, tb=None):\n        try:\n            if value is None:\n                value = tp()\n            if value.__traceback__ is not tb:\n>               raise value.with_traceback(tb)\nE               pytest_assume.plugin.FailedAssumption: \nE               2 Failed Assumptions:\nE               \nE               test_xdist.py:36: AssumptionFailure\nE               >>\tpytest.assume(1==0)\nE               AssertionError: assert False\nE               \nE               test_xdist.py:38: AssumptionFailure\nE               >>\tpytest.assume(False)\nE               AssertionError: assert False\n\nD:\\Miniconda\\Lib\\site-packages\\six.py:718: FailedAssumption"}, "start": 1701430294560, "stop": 1701430294584, "uuid": "90a9ba15-a9ea-4c79-9e26-16fa4b8f6d00", "historyId": "c12f9d12fdee877cf50a1a7455155f69", "testCaseId": "c12f9d12fdee877cf50a1a7455155f69", "fullName": "test_xdist#test_simple_assume7", "labels": [{"name": "suite", "value": "test_xdist"}, {"name": "host", "value": "QuMoSi"}, {"name": "thread", "value": "14004-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_xdist"}]}